test/e2e/tests/hardware-wallets/trezor-sign.spec.ts

```diff
@@ -26,28 +21,13 @@ describe('Trezor Hardware Signatures', function (this: Suite) {
         dapp: true,
       },
       async ({ driver }: { driver: Driver }) => {
-        await unlockWallet(driver);
-
-        await openDapp(driver);
-        await driver.clickElement('#signTypedDataV4');
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-        await driver.delay(1000);
-
-        await driver.clickElementSafe('.confirm-scroll-to-bottom__button');
-        await driver.clickElement({ text: 'Confirm', tag: 'button' });
-        await driver.delay(1000);
-
-        await driver.waitUntilXWindowHandles(2);
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
-        await driver.clickElement('#signTypedDataV4Verify');
-
-        const verifyRecoverAddress = await driver.findElement(
-          '#signTypedDataV4VerifyResult',
-        );
-
-        assert.equal(
-          await verifyRecoverAddress.getText(),
-          KNOWN_PUBLIC_KEY_ADDRESSES[0].address.toLocaleLowerCase(),
+        await loginWithBalanceValidation(driver);
+        const testDappPage = new TestDappPage(driver);
+        await testDappPage.openTestDappPage();
+        await testDappPage.check_pageIsLoaded();
+        await testDappPage.signTypedDataV4(true);
```

Comment:
I think passing an object with the flag name, would make it clearer what this boolean is actually indicating. However given that handling redesign/no-redesign is temporary I think it's fine as it is

Reply:
yeah it's a very temporary solution, let's wait for the removal of the toggle as planned in the ticket https://github.com/MetaMask/MetaMask-planning/issues/3030